types/autobahn-browser/autobahn-browser-tests.ts

```diff
@@ -0,0 +1,76 @@
+import autobahn = require("autobahn-browser");
```

Comment:
Since this is the browser version of autobahn, you should *not* have an import statement, and the tests should still work.